addons/behaviour_toolkit/finite_state_machine/fsm.gd

```diff
@@ -40,7 +38,9 @@ signal state_changed(state: FSMState)
 ## The blackboard of the FSM.
 @export var blackboard: Blackboard
 
-
+## Whether the FSM is active or not.[br]
+## To activate use the [code]start()[/code] method.
+var active: bool = false
```

Comment:
I guess it would make sense to change it to `_active`!

Reply:
I had the same though actually! Then I did it... then I realized some nodes actually need and do call `active`... so it's not only internal property, not sure what to do.

Reply:
Then let's just leave it at that for now.